internal/output/output.go

```diff
@@ -97,9 +97,15 @@ func (p Project) ToSchemaProject() *schema.Project {
 		resources = append(convertOutputResources(p.Breakdown.Resources, false), convertOutputResources(p.Breakdown.FreeResources, true)...)
 	}
 
+	var clonedMetadata *schema.ProjectMetadata
+	if p.Metadata != nil {
+		m := *p.Metadata
+		clonedMetadata = &m
+	}
```

Comment:
what is the importance of cloning the pointer here? Can we add a comment to clarify?